cmd/analyse_test.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/damianoneill/nc-hammer/result"
+)
+
+var ts1 = result.NetconfResult{0, 0, "10.0.0.1", "get", 0, "error", 0}
```

Comment:
Try to use valid values for some of your tests, as users will typically look at the tests for supplemental information.  Run a suite and use the values generated as input for your test.